client/scripts/sw-docker-entrypoint

```diff
@@ -102,9 +81,20 @@ restore_activate_runtime() {
     eval "$(${SWRT_DIR}/activate.sw)" || exit 1
 }
 
+pull_model_and_runtime() {
+    echo '--> pulling model and runtime ...'
+    swcli instance login --token "${SW_TOKEN}" --alias server ${SW_INSTANCE_URI}
+    swcli model copy cloud://server/project/${SW_PROJECT}/model/${SW_MODEL_VERSION} .
+    swcli runtime copy cloud://server/project/${SW_PROJECT}/runtime/${SW_RUNTIME_VERSION} .
+}
+
 run() {
-    echo "--> start to run swmp ${STEP}, use $(which swcli) cli @ ${SWMP_DIR} ..."
-    swcli ${VERBOSE} model eval "${SWMP_DIR}"/src --step=${STEP} --task-index=${TASK_INDEX} --override-task-num=${TASK_NUM} --version=${SW_EVALUATION_VERSION} || exit 1
+    echo "--> start to run evaluation: ${STEP}, use $(which swcli) cli ..."
+    swcli ${VERBOSE} eval run \
+    --step=${STEP} --task-index=${TASK_INDEX} --override-task-num=${TASK_NUM} \
+    --model ${SW_MODEL_VERSION} \
+    --runtime ${SW_RUNTIME_VERSION} \
```

Comment:
in the standalone instance, when users run model eval with `--use-docker` argument, the `--model` and `--runtime` can work right?

Reply:
fixed